dev/tasks/python-wheels/github.linux.yml

```diff
@@ -24,13 +24,16 @@ permissions:
 
 jobs:
   build:
-    name: "Build wheel for manylinux {{ manylinux_version }}"
+    name: "Build wheel for {{ linux_wheel_name }}linux {{ linux_wheel_version }}"
```

Comment:
Same here: would prefer `linux_wheel_kind` and "manylinux"/"musllinux" instead of "many"/"musl".